app/models/item.rb

```diff
@@ -4,5 +4,5 @@ class Item < ApplicationRecord
 
   has_and_belongs_to_many :auth_ip_addresses
   has_and_belongs_to_many :auth_services
-  belongs_to :end_service
+  belongs_to :end_service, optional: true
```

Comment:
@eshadatta Thanks for the catch! Early on for this feature addition, I had confused end_service and auth_service, so I switched end_service to optional in order to allow for ip or auth_service based auth.
Reverted, Thanks!